### ISSUE TITLE
Is Clop reducing the image quality?

### ISSUE THREAD

**Issue by ggrelet** [FuzzyIdeas/Clop#2]
Couldn't find the answer neither here or on your website.

Awesome app, btw!

**Comment by alin23** [comment]
It depends on the image. For example PNG compression is lossless, **but** if you have a PNG in a deeper colorspace than sRGB (like a HDR screenshot or something), then Clop will reduce it to sRGB and the colors can look more faded.

JPEGs are lossy by definition and any further encode will reduce the quality (but Clop tries to keep it 99% similar).

GIFs, well they don't look that good anyway, there's not much you can do to compress and keep their quality the same.

**Comment by ggrelet** [comment]
Thanks for the answer. I'm closing this issue. Maybe consider adding a FAQ to the project?

**Comment by alin23** [comment]
Yep, when I'll have time for it, I'll add an FAQ. This was the first time this question was asked so I had no "frequently" asked questions yet 😅
